bigtable-hbase-integration-tests/src/test/java/com/google/cloud/bigtable/hbase/TestFilters.java

```diff
@@ -344,12 +344,14 @@ public void testRowFilterBinaryComparator() throws Exception {
       table.put(put);
     }
 
-    // Test BinaryComparator - EQUAL
+    Filter filter;
+    Result[] results;
     ByteArrayComparable rowKey2Comparable = new BinaryComparator(rowKey2);
-    Filter filter = new RowFilter(CompareFilter.CompareOp.EQUAL, rowKey2Comparable);
-    Result[] results = scanWithFilter(table, rowKey1, rowKey4, qual, filter);
-    Assert.assertEquals("# results", 1, results.length);
-    Assert.assertArrayEquals(rowKey2, results[0].getRow());
+
+    // Test BinaryComparator - NO_OP
+    filter = new RowFilter(CompareFilter.CompareOp.NO_OP, rowKey2Comparable);
+    results = scanWithFilter(table, rowKey1, rowKey4, qual, filter);
+    Assert.assertEquals("# results", 0, results.length);
```

Comment:
I thought NO_OP is ACCEPT_ALL_FILTER, shouldn't it match all instead of getting size 0?